snapfaas/src/labeled_fs/mod.rs

```diff
@@ -25,10 +26,8 @@ lazy_static::lazy_static! {
 
         let default_db = dbenv.open_db(None).unwrap();
         let mut txn = dbenv.begin_rw_txn().unwrap();
-        let root_uid = get_next_uid();
-        if let Err(lmdb::Error::NotFound) = get_val_db(root_uid, &txn, default_db) {
-            put_val_db(root_uid, Directory::new().to_vec(), &mut txn, default_db);
-        }
+        let root_uid = 0;
+        let _ = put_val_db_no_overwrite(root_uid, Directory::new().to_vec(), &mut txn, default_db);
```

Comment:
This little clause is getting big enough to warrent inline comments describing what it's doing.

Reply:
[comment](https://github.com/princeton-sns/snapfaas/pull/31/commits/2a4bd3c963f1146996ff18b05508858fc1766102) added